### ISSUE TITLE
Display bug: reference counts show up over adjacent panes.

### ISSUE THREAD

**Issue by strout** [OmniSharp/omnisharp-atom#697]
When I have multiple panes open, with my code in the left pane, I see the blue circles containing reference counts on top of the right pane. I would expect them to only be visible after scrolling right far enough that they're in the left pane.

<img width="358" alt="screen shot 2016-03-25 at 1 38 29 pm" src="https://cloud.githubusercontent.com/assets/110026/14051200/1f647520-f28f-11e5-984d-57e268ffff60.png">

In this image, the "21" is a line number from the right pane (a git-plus diff pane). I would expect the "2" in the blue circle not to be visible unless I scroll right far enough that it doesn't overlap with other panes.

**Comment by strout** [comment]
I just noticed that the reference count bubbles also draw over the status bar and other panels at the bottom of the window.

**Comment by david-driscoll** [comment]
Yeah the decorations are kinda crazy in how they show up elsewhere in the
editor.

I'm looking at the block decorations to build something similar to code
lens as you might see in vscode / visual studio.

Then plan will be offer the bubbles as an option going forward ( know some
people like them better!).

**Comment by strout** [comment]
@david-driscoll Great! Block decorations are a good solution to me. (Like you said, though, others might not like them.)
